lib/cli.js

```diff
@@ -17,6 +17,8 @@ Options:
     monitor the file system for changes to recompile automatically
   --fingerprint
     add unique hash to file names
+  --source-maps
+    add inline source maps to files
   --compact
     reduce output size (e.g. via minification)
```

Comment:
Unrelated, but suggesting minification is wrong.

For these kinds of plugin-dependent options we should probably just add "(where supported)".